Packs/Whois/Integrations/Whois/Whois.yml

```diff
@@ -1,19 +1,25 @@
 category: Data Enrichment & Threat Intelligence
 commonfields:
-  id: Whois
+  id: Whois ip
   version: -1
 configuration:
 - display: Return Errors
   name: with_error
   required: false
   type: 8
-- display: Proxy URL. Supports socks4/socks5/http connect proxies (e.g. socks5h://host:1080)
+- display: Proxy URL. (for the Domain and whois commands) Supports socks4/socks5/http connect
+    proxies (e.g. socks5h://host:1080)
   name: proxy_url
   required: false
   type: 0
+- display: Use system proxy settings
+  hidden: false
+  name: proxy
+  required: false
+  type: 8
```

Comment:
What happens if a user sets one URL in the XSOAR system proxy and another URL in the proxy_url param?

Reply:
for the Domain and Whois commands It will use the proxy_url, and in the ip command, it will use the system proxy.